### ISSUE TITLE
TalkBack cannot read content in app created by PhoneGap

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#130]
```
What steps will reproduce the problem?
1. Create a simple PhoneGap Android app
2. Utilize standard accessibility tags
3. Open TalkBack - just says "WEB CONTENT" and that's all

What is the expected output? What do you see instead?
TalkBack should read the content like an accessible web site.

What version of the product are you using? On what operating system?
Android 2.2-4.1.

Please provide any additional information below.
This is a major issue for government and educational institutions that have to 
create accessible apps for Android. I hate to say it, but iOS voiceover works 
great with PhoneGap-created apps.

```

Original issue reported on code.google.com by `rdla...@gmail.com` on 5 Sep 2012 at 2:35

**Comment by GoogleCodeExporter** [comment]
```
Hey, I'm one of the PhoneGap core contributors and my particular area of 
expertise is Android. If there is anything I can do to help with this bug 
please let me know.
```

Original comment by `simon.ma...@gmail.com` on 6 Sep 2012 at 12:48

**Comment by GoogleCodeExporter** [comment]
```
Since PhoneGap utilizes WebViews, you'll need to insure you have web 
accessibility script injection turned on.

ICS: Settings > Accessibility > Install web scripts
Jelly Bean: Settings > Accessibility > Enhance web accessibility

Simon, our recommendation for developers of web applications on Android is to 
follow the standard W3C accessibility standards and our built-in javascript 
screen reader will work with the application's content.
```

Original comment by `caseybur...@google.com` on 13 Sep 2012 at 6:55
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
Thank you so much for taking the time to explain. I was testing on a 2.3.5 
phone which does not have those settings. I see they are available on my ICS 
phone.
```

Original comment by `simon.ma...@gmail.com` on 13 Sep 2012 at 7:21

**Comment by GoogleCodeExporter** [comment]
```
I cannot replicate this functionality on my Xoom tablet running 4.1. If I 
enable "enhance web accessibility" and launch my app, TalkBack only says "web 
content" and that is all. The entire viewport has the that border applied to it 
- no buttons or content are highlighted and nothing else is spoken when I'm 
navigating the app.

The same code wrapped up for iOS is spoken well with VoiceOver, so I doubt that 
the accessible tags I'm using in the HTML are to blame.
```

Original comment by `rdla...@gmail.com` on 14 Sep 2012 at 1:01

**Comment by GoogleCodeExporter** [comment]
```
Content description cannot be read anymore since the latest version of TalkBack.
We are developing our application with custom view in Java and HTML5. The 
problem is still the same. Phonegap plugin is not guilty.
Google/TalkBack developers need to fix this issue as soon as possible. Visually 
impaired users lose essential information in most web application. Please think 
about Android Users with previous version. Thanks.
```

Original comment by `ifeelpi...@gmail.com` on 3 Dec 2012 at 6:17

**Comment by GoogleCodeExporter** [comment]
```
Hi, I am facing same issue. Please let me know if you have any update related 
to it.
Thanks.
```

Original comment by `rajat.in...@gmail.com` on 21 Apr 2013 at 6:54

**Comment by GoogleCodeExporter** [comment]
```
Issue 228 has been merged into this issue.
```

Original comment by `caseybur...@google.com` on 21 Apr 2013 at 9:31

**Comment by GoogleCodeExporter** [comment]
```
Any updates?

It seems TalkBack isn't work for javascript enabled webview in KitKat too.
( Phonegap webview is javascript enabled default. )

And I cant find Enhance web accessibility option in KitKat too.

```

Original comment by `dongse...@gmail.com` on 14 Feb 2014 at 2:10
